use-cases/using-Intel-SGX-and-kata.md

```diff
@@ -0,0 +1,68 @@
+# Kata Containers with SGX
+
+- [Install Host kernel with SGX support](#install-host-kernel-with-sgx-support)
+- [Install Guest kernel with SGX support](#install-guest-kernel-with-sgx-support)
+- [Run Kata Containers with SGX enabled](#run-kata-containers-with-sgx-enabled)
+
+Intel® Software Guard Extensions (SGX) is a set of instructions that increases the security
+of applications code and data, giving them more protections from disclosure or modification.
+At the time of writing this document, SGX patches have not landed on the Linux kernel
```

Comment:
You might want to make this sentence into a ["Note"](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#notes) to make it stand out (for general readers and us, as we will need to update this doc when support is more common.